frontend/containers/project-form-pages/pages/general-information.tsx

```diff
@@ -0,0 +1,275 @@
+            <fieldset name="involved_project_developer">
+              <legend className="inline font-sans font-semibold text-sm text-gray-800 mb-4.5">
+                <span className="mr-2.5">
+                  <FormattedMessage
+                    defaultMessage="Are there other Project developers involved in the project?"
+                    id="Xk8qys"
+                  />
+                </span>
+                <FieldInfo
+                  infoText={formatMessage({
+                    defaultMessage: 'Do you have a partnership with someone else?',
+                    id: 'nbqoY2',
+                  })}
+                />
+              </legend>
+              <Label className="block font-normal">
+                <input
+                  id="involved-project-developer"
+                  type="radio"
+                  value={1}
+                  className="mr-2"
+                  {...register('involved_project_developer')}
+                  onChange={() => setShowInvolvedProjectDevelopers(true)}
+                />
+                <FormattedMessage defaultMessage="Yes" id="a5msuh" />
+              </Label>
+              <Label className="block mt-4 font-normal">
+                <input
+                  id="involved-project-developer"
+                  type="radio"
+                  value={0}
+                  className="mr-2"
+                  {...register('involved_project_developer', {
+                    onChange: () => setShowInvolvedProjectDevelopers(false),
+                  })}
+                />
+                <FormattedMessage defaultMessage="No" id="oUWADl" />
+              </Label>
+              <ErrorMessage
+                id="involved-project-developer-error"
+                errorText={(errors?.involved_project_developer as FieldError)?.message}
+              />
+            </fieldset>
+            {/* Project developers selector */}
+            <div
+              className={cx('relative top-[-66px] left-16 w-[360px]', {
+                hidden: !showInvolvedProjectDevelopers,
+              })}
+            >
+              <Combobox
+                id="involved-project-developer-ids"
+                aria-label={formatMessage({
+                  defaultMessage: 'Project developers involved in the project',
+                  id: 'SYPx2d',
+                })}
+                name="involved_project_developer_ids"
+                control={control}
+                controlOptions={{
+                  disabled: false,
+                }}
+                placeholder={formatMessage({
+                  defaultMessage: 'select project developers',
+                  id: 'QESbGa',
+                })}
+              >
+                {[
+                  ...projectDevelopers,
+                  // The hardcoded option Not Listed
+                  { id: 'not-listed', attributes: { name: 'not listed' } },
```

Comment:
🟠

You need to translate the option. Also, could the first letter be uppercase (“Not listed”)? Thanks!